engine/runtime-language-arrow/src/main/java/org/enso/interpreter/arrow/runtime/ArrowCastToFixedSizeArrayFactory.java

```diff
@@ -12,6 +12,7 @@
 import com.oracle.truffle.api.library.ExportLibrary;
 import com.oracle.truffle.api.library.ExportMessage;
 import java.nio.*;
+import java.util.BitSet;
 import org.apache.arrow.memory.util.MemoryUtil;
```

Comment:
I thought we want an independent implementation of arrow. Depending on Arrow Java library means we are creating a Truffle wrapper, aren't we?

Reply:
I was re-using the code for creating memory-mapped buffers. But you are right that there is no need to bring the whole dependency for that. Fixed